admin/src/main/java/edu/unc/lib/dl/admin/controller/PerformanceMonitorController.java

```diff
@@ -0,0 +1,299 @@
+		CSVPrinter csvFilePrinter = null;
+		Set<String> operations = null;
+		String filePath = "/opt/data/ingest-times-daily.csv";
+		Map<String, String> depositJob = null;
+		Map<String, String> operationJob = null;
+		String[] depositKeys = null;
+
+		if (buildFile(filePath)) {
+			Set<String> deposits = getDepositMetrics();
+			try (Jedis jedis = getJedisPool().getResource()) {
+				operations = jedis.keys("operation-metrics:*");
+
+				fileWriter = new FileWriter(filePath);
+				csvFilePrinter = new CSVPrinter(fileWriter, csvFileFormat);
+				csvFilePrinter.printRecord(FILE_HEADERS);
+
+				Boolean matchingDate = false;
+				for (String deposit : deposits) {
+					depositJob = jedis.hgetAll(deposit);
+					depositKeys = deposit.split(":");
+					
+					// Ignore data for individual deposits by uuid. Only need the daily ones in this instance
+					if (depositKeys.length > 2) {
+						continue;
+					}
+
+					String jobDate = depositKeys[1];
+					String throughputFiles = depositJob.get("throughput-files");
+					String throughputBytes = depositJob.get("throughput-bytes");
+					String finished = depositJob.get("finished");
+					String failed = depositJob.get("failed");
+					String failedDepositJob = depositJob.get("failed-job:edu.unc.lib.dl.cdr.services.techmd.TechnicalMetadataEnhancementService");
+
+					for (String operation : operations) {
+						String operationDate = operation.split(":")[1];
+						
+						if (operationDate.equals(jobDate)) {
+							operationJob = jedis.hgetAll(operation);
+
+							String moves = (operationJob.get("moves"));
+							String finishedEnhancements = operationJob.get("finished-enh:edu.unc.lib.dl.cdr.services.techmd.TechnicalMetadataEnhancementService");
+							String failedEnhancements = operationJob.get("failed-enh:edu.unc.lib.dl.cdr.services.techmd.TechnicalMetadataEnhancementService");
+
+							List<String> data = new ArrayList<>();
+							data.add(jobDate);
+							data.add("N/A");
+							data.add(throughputFiles);
+							data.add(throughputBytes);
+							data.add("0");
+							data.add("0");
+							data.add(finished);
+							data.add(moves);
+							data.add(finishedEnhancements);
+							data.add(failedEnhancements);
+							data.add(failed);
+							data.add(failedDepositJob);
+	
+							csvFilePrinter.printRecord(data);
+	
+							matchingDate = true;
+							break;
+						} else {
+							matchingDate = false;
+						}
+					}
+	
+					if (!matchingDate) {
+						List<String> data = new ArrayList<>();
+						data.add(jobDate);
+						data.add("N/A");
+						data.add(throughputFiles);
+						data.add(throughputBytes);
+						data.add("0");
+						data.add("0");
+						data.add(finished);
+						data.add("0");
+						data.add("0");
+						data.add("0");
+						data.add(failed);
+						data.add(failedDepositJob);
+	
+						csvFilePrinter.printRecord(data);
+					}
+				}
+			} catch (Exception e) {
+				log.error("Failed to write data to {}", filePath, e);
+			} finally {
```

Comment:
If you switch over to opening everything in the try() section you shouldn't need this finally